pkg/controller/miganalytic/restic_executor.go

```diff
@@ -64,7 +65,7 @@ func (r *ResticDFCommandExecutor) DF(podRef *corev1.Pod, persistentVolumes []Mig
 	err := podCommand.Run()
 	if err != nil {
 		log.Info(
-			fmt.Sprintf("df command inside pod %s returned non-zero error code", podRef.Name))
+			fmt.Sprintf("Failed running df command inside pod %s", podRef.Name))
```

Comment:
Do we have any more detailed info we could log here? Err code, stderr, stdout? Or is there somewhere else we're putting this info

Reply:
@djwhatle we do not have any other error info. This is the case when the command itself fails to run for some unknown reason. In that case, all PVCs present on this node will be skipped from PV resizing. The exact error is suppressed. However, if the command runs, we process command's StdErr and StdOut both to determine the error per PV.

Reply:
Yeah I can see that we have the stderr and stdout, that's kind of why I'm asking. How does the user see what went wrong here? Where do they look?

Reply:
@djwhatle The name of the PVs for which the PV resizing failed are shown in a Warning condition on plan validation page. But the exact errors are not shown

Reply:
Some recent discussion with @shawn-hurley and @jortel has focused on need for more informative logs when things go wrong. This feels like somewhere we may want to give info about what went wrong if we have it. I'll leave to your judgement if there's anything reasonable we can give to the user.